components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/cmd/BuildCmd.java

```diff
@@ -61,6 +61,7 @@ public void execute() {
 
         try {
             projectName = GatewayCmdUtils.getProjectName(mainArgs);
+            projectName = projectName.replaceAll("\\/","");
```

Comment:
In windows the default added character would be backward slash (\). So error will continue to occur in windows